pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -239,23 +238,24 @@ func ensureDHCP(bindMechanism BindMechanism, podInterfaceName string) error {
 	return nil
 }
 
-func (l *podNICImpl) PlugPhase2(vmi *v1.VirtualMachineInstance, iface *v1.Interface, network *v1.Network, domain *api.Domain, podInterfaceName string) error {
+func (l *podNICImpl) PlugPhase2(vmi *v1.VirtualMachineInstance, network *v1.Network, domain *api.Domain, podInterfaceName string) error {
 	precond.MustNotBeNil(domain)
 
 	// There is nothing to plug for SR-IOV devices
-	if iface.SRIOV != nil {
+	if l.iface.SRIOV != nil {
 		return nil
 	}
 
-	bindMechanism, err := l.getPhase2Binding(vmi, iface, network, domain, podInterfaceName, nil)
+	bindMechanism, err := l.getPhase2Binding(vmi, &l.iface, network, domain, podInterfaceName, nil)
```

Comment:
move this line just befor the place where "bindMechanism" is first used "bindMechanism.loadCachedVIF"

Reply:
Done.